src/cecs277TermProject/Room.java

```diff
@@ -1,20 +1,24 @@
 package cecs277TermProject;
 
-public abstract class Room {
+public class Room {
     protected int roomNumber;
     protected RoomType type;
-    protected double baseRate;
 
     public Room(int roomNumber, RoomType type) {
         this.roomNumber = roomNumber;
         this.type = type;
     }
 
-
-    public abstract double getCost();
-
     @Override
     public String toString() {
         return String.format("%d", roomNumber);
     }
+
+	public  double getCost() {
+		return 0;
+	}
```

Comment:
If this is just return a dummy and never used, we should declare them as abstract and let its children define the method.

Reply:
Is the price going to vary depending on the room and the type of room it is?
So the cost would be unique to each room type right?

Reply:
nvm just noticed you had subclasses for each type